var/spack/repos/builtin/packages/py-fenics-ffc/package.py

```diff
@@ -33,13 +33,20 @@ class PyFenicsFfc(PythonPackage):
     depends_on("py-setuptools", type=("build", "run"))
     depends_on("py-numpy", type=("build", "run"))
 
-    for ver in ["2019.1.0.post0", "2018.1.0", "2017.2.0.post0", "2016.2.0"]:
+    for ver in ["master", "2019.1.0.post0", "2018.1.0", "2017.2.0.post0", "2016.2.0"]:
```

Comment:
Instead of adding this to the list and adding special conditions for master/not-master, why not leave it out of the list and add one extra line for master for each dep? Or add master versions of these deps?

Reply:
Done in https://github.com/spack/spack/pull/42182/commits/be74f4a809a62d5a7f6028aa77fb309d26d81d56